messaging-experiments.json

```diff
@@ -1391,5 +1391,174 @@
         }
       ]
     }
+  },
+  {
+    "id": "PRIVACY-RETENTION-STUDY-V2-BUG-1656311",
+    "enabled": true,
+    "filter_expression": "(env.channel == 'release' || env.channel == 'default') && env.version >= '79.'",
```

Comment:
do we want to keep env.channel == 'default' ?

Reply:
Good point, we should exclude `default`